docker-compose.yml

```diff
@@ -39,31 +40,57 @@ services:
       - UWSGI_OFFLOAD_THREADS=1
       - UWSGI_HARAKIRI=15
       - UWSGI_DIE_ON_TERM=1
-      - UWSGI_MODULE=settings.wsgi:application
+      - UWSGI_MODULE=main.wsgi:application
+      - UWSGI_PY_AUTORELOAD=1
+
+  test:
+    build:
+      context: .
+      target: tests
+      args:
+        - SECRET_KEY=dev
+        - AUTHORIZATION_TOKEN=auth-token
+        - GET_AUTHORIZATION_TOKEN=get-auth-token
+    volumes:
+      - ./src:/src
+      - ./tests:/tests
+      - ./deploy:/deploy
+    environment:
+      - SECRET_KEY=dev
+      - AUTHORIZATION_TOKEN=auth-token
+      - GET_AUTHORIZATION_TOKEN=get-auth-token
+      - DATABASE_NAME=waarnemingen_mensen
+      - DATABASE_HOST=database
+      - DATABASE_USER=waarnemingen_mensen
+      - DATABASE_PASSWORD=insecure
+      - UWSGI_HTTP=0.0.0.0:8001
+      - UWSGI_MASTER=1
+      - UWSGI_STATIC_INDEX=index.html
+      - UWSGI_STATIC_MAP=/mensen/static=/static
+      - UWSGI_CALLABLE=application
+      - UWSGI_VACUUM=1
+      - UWSGI_STATIC_EXPIRES=/* 3600
+      - UWSGI_OFFLOAD_THREADS=1
+      - UWSGI_HARAKIRI=15
+      - UWSGI_DIE_ON_TERM=1
+      - UWSGI_MODULE=main.wsgi:application
       - UWSGI_PY_AUTORELOAD=1
```

Comment:
De environment vars die je nu meerdere keren gebruikt kan je verplaatsen naar een soort template, en dan hergebruiken.

Reply:
Gezien het er nu nog maar vier vars zijn die herbruikt worden, laat ik ze er nu even in staan. Sowieso kan ik trouwens die uwsgi vars er ook uit halen, maar dat komt later.